include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -1177,6 +1185,19 @@ struct ValueImpl : ConstValueImpl<T> {
   /// <param name="index">[in] Index of the string in the tensor to set</param>
   void FillStringTensorElement(const char* s, size_t index);
 
+  /// <summary>
+  /// Allocate if necessary and obtain a pointer to a UTF-8
+  /// encoded string element buffer indexed by the flat element index,
+  /// and at least the specified buffer length.
+  ///
+  /// This API is for advanced usage. It avoids a need to construct
+  /// an axillary array of string pointers and allows to write data directly (do not zero terminate).
```

Comment:
Assuming no armpits are involved here.

```suggestion
  /// an auxiliary array of string pointers and allows to write data directly (do not zero terminate).
```